ogr/services/gitlab.py

```diff
@@ -398,7 +398,11 @@ def create_release(
         )
 
     def get_latest_release(self) -> GitlabRelease:
-        raise NotImplementedError()
+        releases = self.gitlab_repo.releases.list()
+        return self._release_from_gitlab_object(
+            raw_release=releases[0],
```

Comment:
Are we sure it is the latest?

Reply:
https://docs.gitlab.com/ee/api/releases/index.html#list-releases
> Paginated list of Releases, sorted by released_at.

Reply:
OK, thanks! Maybe add that note to comment.

Reply:
Thanks. I would maybe swap the two lines, but we can leave it as is.